### PR TITLE
Update ssd1680_simpletest.py

### PR DESCRIPTION
**Pull request by dave-ct**
Added comment on how to change display height for Monochrome version to work correctly

### CODE REVIEW

**Review by tannewt — COMMENTED** [review]
Want to add a separate example for the monochrome display?

**Comment by dave-ct** [comment]
> Want to add a separate example for the monochrome display?

@tannewt  Will have a go at the weekend and update this pull request.

**Comment by tannewt** [comment]
Thank you!

**Comment by makermelissa** [comment]
> > Want to add a separate example for the monochrome display?
> 
> @tannewt Will have a go at the weekend and update this pull request.

I think maybe adjusting the offset by adding a `rowstart =5` parameter (possibly it may be `colstart` instead) to the init code would be the appropriate fix rather than adjusting the height which brings its own set of issues.

**Comment by jposada202020** [comment]
probably would be a good idea to verify #5 at the same time. sadly, I do not have the hardware to make the tests.

**Comment by dave-ct** [comment]
@makermelissa @jposada202020 @tannewt Have made an update to the ssd1680 and simpletest example that works now on both mono and tri colour featherwings.  As cannot test on other devices have made clostart default to 1 unless set otherwise so ssd1680 will still work previous simpletest versions.

**Comment by dave-ct** [comment]
Not sure why the Formatting checks have failed?

**Comment by jposada202020** [comment]
dave-ct is because of the formating.

 Error: reformatted adafruit_ssd1680.py from (https://github.com/adafruit/Adafruit_CircuitPython_SSD1680/actions/runs/4187808124/jobs/7258138283)

this is easily fixed using `pre-commit run --all-files` you could find more info here...

https://learn.adafruit.com/improve-your-code-with-pylint/check-your-code-with-pre-commit

Let me know if  you need more help :)

**Comment by dave-ct** [comment]
@jposada202020 that did the job, checks all passed now.  Thanks for the pointer.

**Comment by jposada202020** [comment]
@dave-ct I will take a look over the weekend, hopefully, thanks again :)

**Comment by makermelissa** [comment]
Hi, for the driver, because of the `**kwargs` in the __init__ function, you should be able to just remove the `colstart=1,` and just add `colstart` to the init code in the example and it will automatically pass through. This avoids introducing a new parameter.

Since the code is likely already in use, you should also check if it was in kwargs and default to 1 if not. This way we can have the intended functionality without breaking anything.

File: adafruit_ssd1680.py [hunk 1]
```diff
@@ -95,6 +97,6 @@ def __init__(self, bus: displayio.Fourwire, **kwargs) -> None:
             set_current_column_command=0x4E,
             set_current_row_command=0x4F,
             refresh_display_command=0x20,
-            colstart=1,
+            colstart=column_correction,
```
Comment (makermelissa) [inline adafruit_ssd1680.py:100]:
```suggestion
```

File: adafruit_ssd1680.py [hunk 2]
```diff
@@ -63,7 +65,7 @@ class SSD1680(displayio.EPaperDisplay):
           Display rotation
     """
 
-    def __init__(self, bus: displayio.Fourwire, **kwargs) -> None:
+    def __init__(self, bus: displayio.Fourwire, column_correction=1, **kwargs) -> None:
```
Comment (makermelissa) [inline adafruit_ssd1680.py:68]:
```suggestion
    def __init__(self, bus: displayio.Fourwire, **kwargs) -> None:
        if "colstart" not in kwargs:
            kwargs["colstart"] = 1
```

File: adafruit_ssd1680.py [hunk 3]
```diff
@@ -51,6 +52,7 @@ class SSD1680(displayio.EPaperDisplay):
     r"""SSD1680 driver
 
     :param bus: The data bus the display is on
+    :param column_correction: Adjust colstart, defaults to 1 (``int``)
```
Comment (makermelissa) [inline adafruit_ssd1680.py:55]:
```suggestion
```

File: examples/ssd1680_simpletest.py [hunk 4]
```diff
@@ -30,15 +34,18 @@
 )
 time.sleep(1)
 
+# For issues with display not updating top/bottom rows correctly set column_correction to 8
 display = adafruit_ssd1680.SSD1680(
     display_bus,
+    column_correction=1,
```
Comment (makermelissa) [inline examples/ssd1680_simpletest.py:40]:
```suggestion
    colstart=1,
```

**Review by makermelissa — CHANGES_REQUESTED** [review]
Here's the changes I suggested.

**Comment by dave-ct** [comment]
@makermelissa much cleaner way than what I did.  Tested fine on my devices.

**Review by makermelissa — APPROVED** [review]
Looks great. thanks.

**Comment by tannewt** [comment]
If different devices need a different colstart setting, then it'd be ideal to have separate examples for them. That's easier than reading comments and changing code.

**Comment by makermelissa** [comment]
Oops, yeah. I guess I merged too soon.

**Pull request merged**